app/src/androidTest/java/com/example/compose/MainActivityTest.kt

```diff
@@ -0,0 +1,134 @@
+package com.example.compose
+
+import android.content.Context.MODE_PRIVATE
+import androidx.compose.ui.test.*
```

Comment:
Don't use wildcard imports by default. You can set it in android studio settings to not use wildcard imports

Reply:
why should not use wildcard? 
when we enable `isMinifyEnabled` , then it will remove all codes which we won't need. is it right?

Reply:
Wildcard import increase build time. It's not significant, but if you would work with application containing 1000 classes the overhead gets bigger. You have to build project every time you want to test your changes, so every developer is doing few builds per day. In big projects it has real impact and it affect all developers. 
Minify is totally different - enabling minify increase build time, but reduce size of final APK. For more information and details about minify and code shrinking check https://developer.android.com/studio/build/shrink-code

Reply:
Also using minify in multidex complex application with multiple dependencies might not be free - It might require more customisation than just setting isMinifyEnabled to true

Reply:
I did not notice that it will increase the build time.